features/config/test/src/test/java/org/opennms/features/config/service/util/OpenAPIConfigHelperTest.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2019-2021 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2021 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ ******************************************************************************/
+package org.opennms.features.config.service.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.media.ArraySchema;
+import io.swagger.v3.oas.models.media.ObjectSchema;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+import org.opennms.features.config.dao.api.ConfigItem;
+import org.opennms.features.config.dao.impl.util.ConfigSwaggerConverter;
+import org.opennms.features.config.dao.impl.util.OpenAPIBuilder;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class OpenAPIConfigHelperTest {
+    String prefix = "/rest/cm";
```

Comment:
nit: I saw we use the same prefix in different places. Maybe it makes sense to centralize that so we can change it later easier since it's just in one place?